templates/user.html

```diff
@@ -15,21 +15,25 @@
   <meta property="og:description" content="See all your Open Source contributions in one place."/>
   <!-- Open Graph End -->
 
+  {{if (ne .ResponseType "iframe")}}
   <!-- Twitter Card Start -->
```

Comment:
@lex111 Absolutely. I have. I may think about once I add more client-side filtering options but for now it seems to be a bit overkill to add React since the user profile page is static.